packages/theming/default-theme/src/createDefaultTheme.ts

```diff
@@ -1,34 +1,8 @@
 import { ThemeReference } from '@fluentui-react-native/theme';
 import { defaultFluentDarkTheme, defaultFluentTheme } from './defaultTheme';
 import { Appearance } from 'react-native';
-import { Theme } from '@fluentui-react-native/theme-types';
-
-export type AppearanceOptions = 'light' | 'dark';
```

Comment:
Not sure if moving these is a good idea. I can keep them here, but then moving getCurrentAppearance out of the default-theme package feels weird since it feels like a circular dependency...